websockets/commands.go

```diff
@@ -16,9 +16,10 @@ type Syncer interface {
 }
 
 type CommandError struct {
-	Name    string `json:"error"`
-	Code    int    `json:"error_code"`
-	Message string `json:"error_message"`
+	Name      string `json:"error"`
+	Code      int    `json:"error_code"`
+	Message   string `json:"error_message"`
+	Exception string `json:"error_exception"`
```

Comment:
Is this a new response field from somewhere? Docs?

Reply:
This should be in a separate PR too. Just in case anyone needs to check the commit history to easily see where changes have been added.

Reply:
Removed from that PR. And will be in a different PR.